apps/web/app/app.dub.co/(dashboard)/[slug]/(ee)/program/bounties/[bountyId]/bounty-submission-details-sheet.tsx

```
@@ -0,0 +1,286 @@
+                    </div>
+                  </div>
+                )}
+
+                {Boolean(submission.urls?.length) && (
+                  <div>
+                    <h2 className="text-content-emphasis text-sm font-medium">
+                      URLs
+                    </h2>
+                    <div className="mt-2 flex flex-col gap-2">
+                      {submission.urls?.map((url) => (
+                        <div className="relative">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)